src/main/java/io/github/jbellis/brokk/SessionManager.java

```diff
@@ -35,8 +35,9 @@ public class SessionManager implements AutoCloseable
      * Record representing session metadata for the sessions management system.
      */
     public record SessionInfo(UUID id, String name, long created, long modified) {
-
-        public boolean isModified() {
+        
+        @JsonIgnore
+        public boolean isSessionModified() {
             return created != modified;
         }
```

Comment:
The method was unhappily named (so json property name was equal to the record property), plus I forgot `@JsonIgnore`.